[05] Rock Paper Scissor/Rock Paper Scissor/Rock Paper Scissor/Haptics/HapticManager.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  HapticManager.swift
+//  Rock Paper Scissor
+//
+//  Created by Joe Pham on 2022-08-22.
+//	From this tutorial: https://www.youtube.com/watch?v=DMBcjZjK844
+//
+
+import UIKit
+
+fileprivate final class HapticManager {
+	static let shared: HapticManager = .init()
+	private let feedback: UINotificationFeedbackGenerator = .init()
+	
+	private init() { }
+	
+	func trigger(_ notifType: UINotificationFeedbackGenerator.FeedbackType) {
+		feedback.notificationOccurred(notifType)
+	}
+}
```

Comment:
Marking it `fileprivate` to reduce access to the low level code itself. Good move!